tests/docker-compose.yaml

```diff
@@ -89,6 +115,8 @@ services:
         condition: service_completed_successfully
       archive:
         condition: service_healthy
+      # webap:
+      #   condition: service_healthy
```

Comment:
Can we remove commented lines, or are we bringing them back at some point?

Reply:
I'd like to leave them there for now.  If I do in fact write tests for the webap, we'll need that service.